packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/process-xcpd.test.ts

```diff
@@ -126,3 +129,97 @@ describe("processXCPDResponse", () => {
     expect(response.patientMatch).toBeNull();
   });
 });
+
+describe("convertIheAddressToAddress", () => {
```

Comment:
Since we're here, let's validate other permutations, like:
- postal code w/ string length < 5
- postal code w/ string length = 5
- postal code w/ string length > 5
- postal code w/ empty string
- postal code w/ numbers instead of string
- postal code w/ number 0
- postal code w/ number w/ less than 5 digits
- postal code w/ number w/ 5 digits
- postal code w/ number w/ more than 5 digits
- etc.